tests/performance/test_data_analysis/statistical_data_analyzer.py

```diff
@@ -0,0 +1,274 @@
+    in the incoming YML file.
+
+    Attributes:
+        baseline_path: path to the CSV file containing the baseline data.
+        datasource_path: path to the CSV file containing the data to be compared.
+        items_path: path to the YML file containing the processes, metrics, and statistics
+        to be analyzed.
+        baseline: Dataframe generated from the file in baseline_path.
+        datasource: Dataframe generated from the file in datasource_path.
+        process_name: name of the process to be analyzed. This will be the name of the related
+        CSV file column. It must be specified in the items_path file.
+        processes: the different processes included in the 'process_name' column of the CSV.
+        They can be several or just one.
+        metrics: metrics to be analyzed. These must be specified in the file items_path.
+    """
+
+    def __init__(self, baseline_path: str, datasource_path: str, items_path: str) -> None:
+        """Initializes the DataLoader.
+
+        Args:
+            baseline_path (str): path to the CSV file containing the baseline data.
+            datasource_path (str): path to the CSV file containing the data to be compared.
+            items_path (str): path to the YML file containing the processes, metrics, and statistics
+            to be analyzed.
+        """
+        self.baseline_path = baseline_path
+        self.datasource_path = datasource_path
+        self.items_path = items_path
+        self.validate_paths()
+        self.baseline = self.load_dataframe(baseline_path)
+        self.datasource = self.load_dataframe(datasource_path)
+        self.process_name, self.processes, self.metrics = self.load_yaml_items(self.items_path)
+
+    def validate_paths(self) -> None:
+        """Validates the existence of the files used by the module."""
+        if not os.path.exists(self.baseline_path) or not os.path.exists(self.datasource_path):
+            raise ValueError("One or both of the provided CSV files do not exist")
+
+        if not os.path.exists(self.items_path):
+            raise ValueError("The YML file does not exit")
+
+    def load_dataframe(self, csv_path: str) -> pd.DataFrame:
+        """Read the CSV and convert it to dataframe. Also check that the format is valid (CSV)
+        and that the file is not empty.
+
+        Args:
+            csv_path (str): path to the CSV file to be converted in Dataframe.
+
+        Returns:
+            dataframe (pd.Dataframe): Dataframe corresponding to the CSV.
+        """
+        dataframe = pd.read_csv(csv_path)
+        if len(dataframe) == 0:
+            raise ValueError(f"The file {csv_path} has not data rows or it has not CSV format")
+
+        return dataframe
+
+    def load_yaml_items(self, yaml_path: str) -> Tuple[str, List[str], Dict[str, Dict[str, float]]]:
+        """Process the YML file containing the elements to be analyzed during the test. In addition,
```

Comment:
Same here, the first line should be a summary. Detailed information should be in a description section, separated by one line

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5622/commits/a8be0593badbbd04bb3211468f64ad2689d17bf3